problems/arraysWithLoops.js

```diff
@@ -66,15 +122,39 @@ function findSmallest() {}
  * @returns {number} second smallest value
  */
 
-function findSecondSmallest() {}
+function findSecondSmallest(nums) {
+  let smallestNum = nums[0]
```

Comment:
This function looks good. 
Try setting your starting variables to Infinity to ensure that you're hitting the right smallest numbers